docs/core/extensions/httpclient-http3.md

```diff
@@ -0,0 +1,116 @@
+- `HttpRequestMessage.VersionPolicy` to `HttpVersionPolicy.RequestVersionOrHigher`.
+
+The reason for requiring a configuration flag for HTTP/3 is to protect apps from future breakage when using version policy `RequestVersionOrHigher`. When calling a server that currently uses HTTP/1.1 and HTTP/2, if the server later upgrades to HTTP/3, the client would try to use HTTP/3 and potentially be incompatible as the standard is not final and therefore may change after .NET 6 is released.
+
+### Platform Dependencies
+
+HTTP/3 uses QUIC as its transport protocol. The .NET implementation of HTTP/3 uses [MsQuic](https://github.com/microsoft/msquic) to provide QUIC functionality. MsQuic is included in specific builds of windows and as a library for Linux. If the platform that HttpClient is running on doesn't have all the requirements for HTTP/3 then it's disabled.
+
+**Windows**
+
+- Windows 11 Build 22000 or later.
+- TLS 1.3 or later connection.
+
+The preceding Windows 11 Build versions may require the use of a [Windows Insider](https://insider.windows.com/) build.
+
+**Linux**
+
+On Linux, libmsquic is published via Microsoft official Linux package repository packages.microsoft.com. In order to consume it, it must be added manually. See [Linux Software Repository for Microsoft Products](https://windows-server/administration/linux-package-repository-for-microsoft-software). After configuring the package feed, it can be installed via the package manager of your distro, for example, for Ubuntu:
+
+```shell
+sudo apt install libmsquic
+```
+
+**macOS**
+
+HTTP/3 is not currently supported on macOS but may be available in a future release.
+
+### Get started
+
+Include the following in the project file to enable HTTP/3 with HttpClient:
+
+```xml
+<ItemGroup>
+    <RuntimeHostConfigurationOption Include="System.Net.SocketsHttpHandler.Http3Support" Value="true" />
+</ItemGroup>
+```
+
+HTTP3 needs to be specified as the version for the request:
+
+```c#
+// See https://aka.ms/new-console-template for more information
+using System.Net;
+
+//Needed to ignore certificate errors if working with local servers
+var handler = new HttpClientHandler()
+{
+    ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator;
+};
+
+var client = new HttpClient(handler);
+client.DefaultRequestVersion =  HttpVersion.Version30;
+client.DefaultVersionPolicy = HttpVersionPolicy.RequestVersionExcat;
+
+Console.WriteLine("--- Localhost:5001 ---");
+var resp = await client.GetAsync("https://localhost:5001/");
+var body = await resp.Content.ReadAsStringAsync();
+Console.WriteLine($"status: {resp.StatusCode}, version: {resp.Version}, body: {body.Substring(0, Math.Min(100, body.Length))}");
+```
+
+### HTTP/3 Server
+
+HTTP/3 is supported by ASP.NET with the Kestrel server in .NET 6. For more details see  [use HTTP/3 with the ASP.NET Core Kestrel web server](/aspnet/core/fundamentals/servers/kestrel/http3).
```

Comment:
Also, this could probably use your `[http3Kestrel]` link you defined have at the bottom of the article.